### PR TITLE
Fixes #16024: Make a plugin from the current OpenSCAP technique

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/16048

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

File: ansible-report/README.asciidoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Rudder plugin: ansible_report
```
Comment (amousset) [inline ansible-report/README.asciidoc]: File extension needs to be .adoc

**Comment by Fdall** [comment]
PR updated with a new commit

File: ansible-report/packaging/prerm [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+NAME=ansible_report
+FOLDERS="ncf/50_techniques techniques dsc/ncf/50_techniques techniques/ncf_techniques"
+cd /var/rudder/configuration-repository/
+git reset
+for folder in $FOLDERS
+do
+  git rm -r $folder/$NAME
+done
+git commit -m "Ansible-report plugin uninstallation"
+# need to be run 2 times to be updated, first one will fail
```
Comment (amousset) [inline ansible-report/packaging/prerm]: Do you have any idea why?

Comment (Fdall) [reply ansible-report/packaging/prerm]: Not really the error given is the following:

+ rudder server reload-techniques
error: Could not reload techniques from files
Error: An error occured when updating the Technique library from file system
cause:Error when trying to delete category with ID 'ActiveTechniqueCategoryId(ansible_report)'
cause:Can not delete 'techniqueCategoryId=ansible_report,techniqueCategoryId=Active Techniques,ou=Rudder,cn=rudder-configuration': subordinate objects must be deleted first

Comment (amousset) [reply ansible-report/packaging/prerm]: Ok, could you open a ticket and add the link in a comment with a TODO to remove it?

File: openscap-report/packaging/postinst [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -x
+
+# Import Ansible Technique
+FOLDERS="ncf/50_techniques/plugin_openscap_report techniques/plugin_openscap_report techniques/ncf_techniques"
```
Comment (amousset) [inline openscap-report/packaging/postinst]: why committing the whole `techniques/ncf_techniques` folder?

**Pull request merged**